pkg/cmd/dev/test.go

```diff
@@ -73,16 +74,21 @@ pkg/kv/kvserver:kvserver_test) instead.`,
 	// or only on failures. `--show-logs` by contrast is a flag for the process
 	// under test, controlling whether the process-internal logs are made
 	// visible.
-	testCmd.Flags().BoolP(vFlag, "v", false, "show testing process output")
+	testCmd.Flags().BoolP(vFlag, "v", false, "show testing process output ("+
+		"equivalent to passing '--testargs='test.v')")
 	testCmd.Flags().Int(countFlag, 1, "run test the given number of times")
 	testCmd.Flags().BoolP(showLogsFlag, "", false, "show crdb logs in-line")
 	testCmd.Flags().Bool(stressFlag, false, "run tests under stress")
-	testCmd.Flags().String(stressArgsFlag, "", "additional arguments to pass to stress")
+	testCmd.Flags().String(stressArgsFlag, "", "additional arguments to pass to stress.\n"+
+		"Prefix each arg with '-'; e.g. '--stress-args='-maxtime=30s'.\n"+
```

Comment:
Nit: An example with multiple arguments would be really helpful. e.g. `--stress-args='-maxtime=30s -maxruns=100'`

Reply:
Done.